codespeed.conf

```diff
@@ -713,6 +739,16 @@ experiments:
         executions:
             - SOMns-interp
 
+    SOMns-Mutex-exp:
+        description: Run mutex benchmarks
+        executions:
+            - SOMns-interp-tn:
+                suites:
+                  - mutex
+            - SOMns-graal-tn:
+                suites:
+                  - mutex
+
```

Comment:
Aren't those same benchmarks already ran a few lines above (see SOMns-Mutex)?

Reply:
the exp stuff was for the CI benchmarking in brussels, with results reported to the speed website. I guess that was deprecated by stefan and I should change it.

Reply:
Yeah, I removed the `exp` duplication, with the data going to ReBenchDB, that's not needed any longer.